<launch>
    <!-- Navigation Program   -->
    <let name="distance_laser" value="0.4"/>
    <let name="speed_factor" value="1.0"/>
    <let name="forward_speed" value="0.2"/>
    <let name="backward_speed" value="-0.2"/>
    <let name="rotation_speed" value="0.5"/>
    <let name="time_to_stop" value="10.0"/>

    <node
        pkg="my_robot_control" exec="my_robot_selfcontrol_node" name="robot_selfcontrol" output="screen">
            <param name="distance_laser" value="$(var distance_laser)"/>
            <param name="speed_factor" value="$(var speed_factor)"/>
            <param name="forward_speed" value="$(var forward_speed)"/>
            <param name="backward_speed" value="$(var backward_speed)"/>
            <param name="rotation_speed" value="$(var rotation_speed)"/>
            <param name="time_to_stop" value="$(var time_to_stop)"/>
    </node>
</launch>